scripts/jenkins/qa_devstack.sh

```diff
@@ -242,4 +244,14 @@ tox -e smoke
 EOF
 fi
 
+h_echo_header "Store tempest results"
+pip install junitxml
+sudo -u stack -i <<EOF
+cd /opt/stack
+if [ -f devstack.subunit ]; then
+    subunit2html devstack.subunit
```

Comment:
Does this need to be output somewhere?

Reply:
I couldn't find any documentation but it creates a file in the current folder with the name results.html automatically

Reply:
:+1: